src/ol/array.js

```diff
@@ -196,6 +196,7 @@ export function equals(arr1, arr2) {
 /**
  * @param {Array<*>} arr The array to sort (modifies original).
  * @param {Function} compareFnc Comparison function.
+ * @api
```

Comment:
This should have some documentation if it is going to be part of the API.

Reply:
OK, I added a description and specified the signature of the comparison function.